Bottom: 7a52d19d54612f5a357239f8b93a7d47bf76751e
Top:    554a968418e246f9f3bb85aad8381468e856955a
Author: Jason Wang <jasowang@redhat.com>
Date:   2016-12-19 17:21:49 +0800

Refresh of xdp-introduce-xdp_hold

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 10764f8..5142e9c 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -405,7 +405,7 @@ static u32 do_xdp_prog(struct virtnet_info *vi,
 	xdp.ring = rq;
 	xdp.private = page;
 	xdp.free = virtnet_xdp_buff_free;
-	xdp.netdev = vi->dev;
+	xdp.dev = vi->dev;
 
 	act = bpf_prog_run_xdp(xdp_prog, &xdp);
 	switch (act) {
